packages/lit-dev-content/site/docs/libraries/bundles.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Bundles
+eleventyNavigation:
+  key: Bundles
+  parent: Related libraries
+  order: 4
+---
+
+Lit is available as pre-built, single-file bundles. These bundles are useful for
+situations where you can't use npm or just want to experiment without setting up
+a build process.
+
+<div class="alert alert-warning">
+
+**If you're using npm for client-side dependencies, you should use the [`lit`
+package](https://www.npmjs.com/package/lit) instead of these bundles.** The
+bundles intentionally combine most or all of Lit into a single file. This can
```

Comment:
It's not just build tools, it's also the browser - the browser doesn't load what's not imported. I would cut this to "This can cause your page to load more JavaScript than needed"